### PR TITLE
[Audit Fix][L-03] Missing docstrings

### PR DESCRIPTION
**Pull request by aleone**
# Task:  L-03 - Missing docstrings

**Should be merged after #758**

## Description

Fixes missing natspec and docstrings in contracts

Fixes ENG-1016

### CODE REVIEW

File: packages/bull-vault/src/AuctionBull.sol [hunk 1]
```diff
@@ -428,7 +454,7 @@ contract AuctionBull is UniFlash, Ownable, EIP712 {
      * @param _orders list of orders
      * @param remainingAmount amount of wPowerPerp to trade
      * @param _clearingPrice clearing price weth/wPowerPerp, in 1e18 units
-     * @param _isDepositingInCrab true if the rebalance will deposit into Crab, false if withdrawing funds from crab
+     * @param _isDepositingInCrab true if the rebalance will deposit into Crab, false if withdrawing funds from Crab
```
Comment (alpinechicken) [inline packages/bull-vault/src/AuctionBull.sol:457]: using lower case 'crab' elsewhere

File: packages/bull-vault/src/BullStrategy.sol [hunk 2]
```diff
@@ -140,7 +143,7 @@ contract BullStrategy is ERC20, LeverageBull {
     }
 
     /**
-     * @notice deposit to crab: deposits crab and ETH, receives USDC and wPowerPerp
+     * @notice deposit to crab: deposits crab and ETH, receives USDC, wPowerPerp and Bull token
```
Comment (alpinechicken) [inline packages/bull-vault/src/BullStrategy.sol:146]: Controversial lack of oxford comma

"The wombat eats shoots, roots, and leaves."

**Review by alpinechicken — APPROVED** [review]
lgtm except for suspicious comma usage

**Review by haythem96 — APPROVED** [review]

File: packages/bull-vault/src/BullStrategy.sol (hunk 2 above)
Comment (aleone) [reply packages/bull-vault/src/BullStrategy.sol:146]: I'm a casual comma user, non of that snobby comma stuff

File: packages/bull-vault/src/AuctionBull.sol (hunk 1 above)
Comment (aleone) [reply packages/bull-vault/src/AuctionBull.sol:457]: fixed

**Pull request merged**